L1Trigger/TrackFindingTracklet/plugins/ProducerKFout.cc

```diff
@@ -58,6 +60,45 @@ namespace trackFindingTracklet {
     const Setup* setup_;
     // helper class to extract structured data from TTDTC::Frames
     const DataFormats* dataFormats_;
+    // Cot bins used to convert from cot to TanL
+    vector<int> cotBins;
+    // Factors used to convert between phi/Z at a radius T, modified to include scale conversions needed in calculation
+    double modChosenRofZ = 0;
+    double modChosenRofPhi = 0;
+    // Corrections to Phi depending on which phi sector
+    int BaseSectorCorr = 0;
+    int UnsignedBaseSector = 0;
+    // Redefine these here to avoid impossible to read code later
+    // Widths used for each part of data stored in the KF track and KF stubs
+    int wTm = 0;
+    int wTSp = 0;
+    int wTSe = 0;
+    int wTphi = 0;
+    int wTinvR = 0;
+    int wTcot = 0;
+    int wTz = 0;
+
+    int wSr = 0;
+    int wSphi = 0;
+    int wSz  = 0;
+    int wSdPhi = 0;
+    int wSdZ = 0;
+
+    int chi2rphiConv = 0;
+    int chi2rzConv = 0;
+    // Bins for dPhi/dZ use to access weight LUT below
+    vector<int> dPhiBins;
+    vector<int> dZBins;
+    // LUT for weighting functions for chi2 calculation
+    vector<int> v0Bins;
+    vector<int> v1Bins;
+    // Bins for final Chi2 Packing
+    vector<double> chi2rphiBins;
+    vector<double> chi2rzBins;
+
+    int chi2ScaleFactor;
+
+    int maxTracksPerEvent;
```

Comment:
Most of these class member will be obsolete as soon as you use the TrackKF and StubKF class, for those which will remain: Please add an underscore at the and of the variable name.